scripts/globals/mobskills/Dread_Wind.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------
+--  Stormwind
+--
+--  Description: Creates a whirlwind that deals Wind damage to targets in an area of effect.
+--  Type: Magical
+--  Utsusemi/Blink absorb: Wipes shadows
+--  Range: Unknown radial
+--  Notes:
+---------------------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+
+---------------------------------------------
+-- OnMobSkillCheck
+---------------------------------------------
+
+function OnMobSkillCheck(target,mob,skill)
+	return 0;
+end;
+
+---------------------------------------------
+-- OnMobWeaponSkill
+---------------------------------------------
+
+function OnMobWeaponSkill(target, mob, skill)
+	local dmgmod = 1;
+
+	local info = MobMagicalMove(mob,target,skill,mob:getWeaponDmg()*3,ELE_WIND,dmgmod,TP_NO_EFFECT);
+	local dmg = MobFinalAdjustments(info.dmg,mob,skill,target,MOBSKILL_MAGICAL,MOBPARAM_WIND,MOBPARAM_WIPE_SHADOWS);
+	
+		if (mob:actionQueueAbility() == true) then		
```

Comment:
none of the actionQueueAbility or ExtraVar stuff is probably going to be used for this.
if it is, set/getExtraVar don't exist anymore and should probably be fixed to use get/setLocalVar